catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -24,23 +26,29 @@
 
     public static final String XML_BOUNDS_TYPE = "GM_Envelope";
 
+    public static final String REGISTRY_ID_PROPERTY = "org.codice.ddf.registry.registry-id";
```

Comment:
Maybe `org.codice.ddf.registry.local-id` or `org.codice.ddf.registry.identity-id`.  That is if I am understanding what is in this system property.